var/spack/repos/builtin/packages/evtgen/package.py

```diff
@@ -54,6 +55,12 @@ def cmake_args(self):
 
         return args
 
+    def patch(self):
```

Comment:
Actually could you add a comment here why this is needed? This is not really self-explanatory

Reply:
@vvolkl this comes from CMS experiment, it is needed for OS X build according to comments.

Reply:
Could you add those comments in the recipe? I think it will help with maintenance.

Reply:
Done